### PR TITLE
Rename `chain` label to `base`; add `chained` label for chained PRs (#4105)

### PR DESCRIPTION
**Pull request by achave11**
Connected issue: #4105 

### Author

- [x] Target branch is `develop`
- [x] Source branch is `issues/<github-handle>/<issue#>-<slug>`
- [x] PR title references the connected issue
- [x] PR title matches<sup>1</sup> title of connected issue         <sub>or comment in PR explains why they're different</sub>
- [x] Title of at least one commit references connected issue
- [x] PR is connected to issue via Zenhub 
- [x] PR description links to connected issue
- [x] Added `partial` label to PR                                    <sub>or this PR completely resolves the connected issue</sub>

<sup>1</sup> when the issue title describes a problem, the PR title is `Fix: ` followed by the issue title   

### Author (reindex)

- [x] Added `r` tag to commit title                                 <sub>or this PR does not require reindexing</sub>
- [x] Added `reindex` label to PR                                   <sub>or this PR does not require reindexing</sub>

### Author (chains)

- [x] This PR is blocked by previous PR in the chain                <sub>or this PR is not chained to another PR</sub>
- [x] Added `chain` label to the blocking PR                        <sub>or this PR is not chained to another PR</sub>

### Author (upgrading)

- [x] Documented upgrading of deployments in UPGRADING.rst          <sub>or this PR does not require upgrading</sub>
- [x] Added `u` tag to commit title                                 <sub>or this PR does not require upgrading</sub>
- [x] Added `upgrade` label to PR                                   <sub>or this PR does not require upgrading</sub>

### Author (operator tasks)

- [x] Added checklist items for additional operator tasks           <sub>or this PR does not require additional tasks</sub>

### Author (hotfixes)

- [x] Added `F` tag to main commit title                            <sub>or this PR does not include permanent fix for a temporary hotfix</sub>
- [x] Reverted the temporary hotfix connected to the issue          <sub>or there is no temporary hotfix for the connected issue on the `prod` branch</sub>

### Author (requirements)

- [x] Ran `make requirements_update`                                <sub>or this PR does not touch requirements*.txt, common.mk, Makefile and Dockerfile</sub>
- [x] Added `R` tag to commit title                                 <sub>or this PR does not touch requirements*.txt</sub>
- [x] Added `reqs` label to PR                                      <sub>or this PR does not touch requirements*.txt</sub>

### Author (rebasing, integration test)

- [x] `make integration_test` passes in personal deployment         <sub>or this PR does not touch functionality that could break the IT</sub>
- [x] Rebased branch on `develop`, squashed old fixups

### Primary reviewer (after rejection)

Uncheck the *Author (requirements)* and *Author (rebasing, integration test)* 
checklists.

### Primary reviewer (after approval)

- [x] Actually approved the PR
- [x] Labeled connected issue as `demo` or `no demo`
- [x] Commented on connected issue about demo expectations          <sub>or labelled connected issue as `no demo`</sub>
- [x] Decided if PR can be labeled `no sandbox`
- [x] PR title is appropriate as title of merge commit
- [x] Ticket is in *Review requested* column
- [x] Requested review from peer reviewer
- [x] Assigned PR to peer reviewer

### Peer reviewer (after rejection)

Uncheck the *Author (requirements)* and *Author (rebasing, integration test)* 
checklists.

### Peer reviewer (after approval)

- [x] `N reviews` label on PR reflects prior peer reviews with changes requested
- [x] Moved ticket to *Approved* column
- [x] Assigned PR to current operator

### Operator (before pushing merge the commit)

- [x] Rename `chain` label to `base`; add `chained` label for chained PRs
- [x] Sync tags to `sync-github-labels`
- [x] Sync tags to `hca-import-validation`
- [x] Sync tags to `hca-metadata-api`
- [x] Checked `reindex` label and `r` commit title tag
- [x] Checked that demo expectations are clear                      <sub>or connected issue is labeled as `no demo`</sub>
- [x] Rebased and squashed branch
- [x] Sanity-checked history
- [x] Pushed PR branch to GitHub
- [x] Branch pushed to GitLab and added `sandbox` label             <sub>or PR is labeled `no sandbox`</sub>
- [x] Build passed in sandbox                                       <sub>or PR is labeled `no sandbox`</sub>
- [x] Deleted unreferenced indices in `sandbox`                     <sub>or this PR does not remove catalogs or otherwise causes unreferenced indices</sub> 
- [x] Started reindex in `sandbox`                                  <sub>or this PR does not require reindexing `sandbox`</sub>
- [x] Checked for failures in `sandbox`                             <sub>or this PR does not require reindexing `sandbox`</sub>
- [x] Added PR reference to merge commit title
- [x] Collected commit title tags in merge commit title
- [x] Moved connected issue to Merged column
- [x] Pushed merge commit to GitHub

### Operator (after pushing the merge commit)

- [ ] Shortened the PR chain                                        <sub>or this PR is not labeled `chain`</sub>
- [ ] Pushed merge commit to GitLab                                 <sub>or merge commit can be pushed later, with another PR</sub>
- [ ] Deleted PR branch from GitHub and GitLab
- [ ] Build passes on GitLab

### Operator (reindex) 

- [ ] Deleted unreferenced indices in `dev`                         <sub>or this PR does not remove catalogs or otherwise causes unreferenced indices</sub> 
- [ ] Started reindex in `dev`                                      <sub>or this PR does not require reindexing</sub>
- [ ] Checked for and triaged indexing failures                     <sub>or this PR does not require reindexing</sub>
- [ ] Emptied fail queues in target deployment                      <sub>or this PR does not require reindexing</sub>

### Operator

- [ ] Unassigned PR

## Shorthand for review comments

- `L` line is too long
- `W` line wrapping is wrong
- `Q` bad quotes
- `F` other formatting problem

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -33,7 +33,8 @@ Connected issue: #0000
 ### Author (chains)
 
 - [ ] This PR is blocked by previous PR in the chain                <sub>or this PR is not chained to another PR</sub>
-- [ ] Added `chain` label to the blocking PR                        <sub>or this PR is not chained to another PR</sub>
+- [ ] Added `base` label to the blocking PR                        <sub>or this PR is not chained to another PR</sub>
+- [ ] Added `chained` label to the blocked PR                <sub>or this PR is not chained to another PR</sub>
```
Comment (hannes-ucsc) [inline .github/pull_request_template.md:37]:
```suggestion
- [ ] Added `chained` label to this PR                <sub>or this PR is not chained to another PR</sub>
```

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
There are other references in the source to the `chain` label that need to be updated.

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -33,7 +33,8 @@ Connected issue: #0000
 ### Author (chains)
 
 - [ ] This PR is blocked by previous PR in the chain                <sub>or this PR is not chained to another PR</sub>
```
Comment (hannes-ucsc) [inline .github/pull_request_template.md:35]: align `<sub>`

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -33,7 +33,8 @@ Connected issue: #0000
...
-- [ ] Added `chain` label to the blocking PR                        <sub>or this PR is not chained to another PR</sub>
+- [ ] Added `base` label to the blocking PR                        <sub>or this PR is not chained to another PR</sub>
```
Comment (hannes-ucsc) [inline .github/pull_request_template.md:36]: align `<sub>`

File: OPERATOR.rst [hunk 4]
```diff
@@ -232,7 +232,7 @@ For an example of how to document failures within a PR `click here`_.
 Chain shortening
 ^^^^^^^^^^^^^^^^
 
-For the blocked PR, change the target branch to develop. Remove the chain
+For the blocked PR, change the target branch to develop. Remove the base
```
Comment (hannes-ucsc) [inline OPERATOR.rst:235]:
```suggestion
For the blocked PR, change the target branch to develop. Remove the ``base``
```

File: OPERATOR.rst [hunk 5]
```diff
@@ -232,7 +232,7 @@ For an example of how to document failures within a PR `click here`_.
...
 label from blocking PR. Last, remove the blocking relationship.
```
Comment (hannes-ucsc) [inline OPERATOR.rst:236]: What about removing the ``chained`` label?

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]

File: OPERATOR.rst [hunk 6]
```diff
@@ -232,8 +232,9 @@ For an example of how to document failures within a PR `click here`_.
 Chain shortening
 ^^^^^^^^^^^^^^^^
 
-For the blocked PR, change the target branch to develop. Remove the chain
-label from blocking PR. Last, remove the blocking relationship.
+For the blocked PR, change the target branch to develop and remove the ``chained``
+label. Remove the ``base`` label from blocking PR. Last, remove the blocking
+relationship.
```
Comment (hannes-ucsc) [inline OPERATOR.rst:237]:
```suggestion
Change the target branch of the blocked PR to ``develop`` and remove the ``chained``
label from that PR. Remove the ``base`` label from the blocking PR. Lastly, remove the blocking
relationship.
```

**Review by hannes-ucsc — CHANGES_REQUESTED** [review]
```diff
Index: CONTRIBUTING.rst
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CONTRIBUTING.rst b/CONTRIBUTING.rst
--- a/CONTRIBUTING.rst	(revision a2e51d654806f8ec5ed9baf401faa10a5eaf71d4)
+++ b/CONTRIBUTING.rst	(date 1651195758779)
@@ -1112,6 +1112,9 @@
   and the target branch setting allows reviewers to ignore changes in the base
   branch.
 
+  Note that in chains involving more than two PRs, the intermediate PRs carry
+  both the ``chain`` and ``base`` labels.
+
 * Rebasing a chained PR involves rebasing its branch on the base branch, instead
   of ``develop``.
 
@@ -1122,11 +1125,6 @@
   where ``start_commit`` is the first commit in ``issues/joe/1234-foo`` that
   wasn't also on the base PR's branch.
 
-* Travis does not build chained PRs by default. To fix this, modify
-  ``branches.only`` in ``.travis.yml`` to list the name of the base branch instead
-  of ``develop``. Commit that change with a title starting in ``drop!``. After
-  the base PR lands, remove the ``drop`` commit.
-
 .. _slug: https://en.wikipedia.org/wiki/Clean_URL#Slug
   
 
```

**Review by hannes-ucsc — APPROVED** [review]

File: CONTRIBUTING.rst [hunk 7]
```diff
@@ -1101,15 +1101,20 @@ Chained PRs
...
+
+  5) In ZenHub, mark PR ``#4`` as blocked by PR ``#3``
 
   This allows the primary reviewer to break the chain when they merge ``#3``.
   The label catches their attention, the dependency lets them follow the chain
   and the target branch setting allows reviewers to ignore changes in the base
   branch.
 
+  Note that in chains involving more than two PRs, the intermediate PRs carry
+  both the ``chain`` and ``base`` labels.
```
Comment (noah-aviel-dove) [inline CONTRIBUTING.rst:1116]:
```suggestion
  both the ``chained`` and ``base`` labels.
```

**Review by noah-aviel-dove — CHANGES_REQUESTED** [review]

**Review by noah-aviel-dove — APPROVED** [review]

**Pull request merged**